packages/p/pkgconf/xmake.lua

```diff
@@ -0,0 +1,29 @@
+package("pkgconf")
+    set_kind("binary")
+    set_homepage("http://pkgconf.org")
+    set_description("A program which helps to configure compiler and linker flags for development frameworks.")
+    add_urls("https://distfiles.dereferenced.org/pkgconf/pkgconf-$(version).tar.xz")
+    add_versions("1.1.0", "5f1ef65d73a880fa5e7012102a17f7b32010e5e46139aed85851a541ba828a63")
+
+    on_load("windows", function(package)
+        package:add("deps", "cmake")
+    end)
+
+    on_install("linux", "bsd", function(package)
+        local configs = {}
+        table.insert(configs, "--enable-shared=" .. (package:config("shared") and "yes" or "no")) 
+        table.insert(configs, "--enable-static=" .. (package:config("shared") and "no" or "yes")) 
+        if package:config("pic") ~= false then
+            table.insert(configs, "--with-pic")
+        end
+
+        import("package.tools.autoconf").install(package, configs)
+    end)
+
+    on_install("windows", function(package)
+        import("package.tools.cmake").install(package)
+    end)
```

Comment:
does pkgconf support windows?

Reply:
https://github.com/pkgconf/pkgconf/blob/master/README.md#compiling-pkgconf-and-libpkgconf-with-cmake-usually-for-windows

It says that it does, but I think it's README is outdated. It doesn't have any CMake stuff.
However I do see meson stuff, so maybe they switched build systems and forgot to document it?

Reply:
Also turns out this is an old pkgconf version because their website is outdated and the download link points to a really old version.

Reply:
And yep, building on Windows was moved to Meson. So I'll update that.